src/components/teaser-hero/readme.md

```diff
@@ -30,6 +30,10 @@ The description text is wrapped into an `p` element to guarantee the semantic me
 
 Avoid slotting block elements (e.g. `div`) as this violates semantic rules and can have negative effects on screen readers.
 
+## Animation
```

Comment:
There is no corresponding code change?

Reply:
It is affected by the changes in the common `panel.scss`
Anyway, this now depends from the decision we take in the previous feedback